### ISSUE TITLE
PanGesture sample does not pan

### ISSUE THREAD

**Issue by sschaub** [jsuarezruiz/xamarin-forms-wpf-samples#1]
Repro steps:

1. Run PanGesture.WPF sample.
2. Attempt to drag image.

Expected: Image moves.
Actual: Image does not move.

I tested on the following platform:

Windows 10.0.17134.165
Visual Studio 15.8.1
.NET Framework 4.7.2

**Comment by jsuarezruiz** [comment]
Yes, it is just one of the pending features to be implemented in the Xamarin.Forms WPF backend. I hope it can be implemented soon and notify you after updating samples.

**Comment by sschaub** [comment]
Thanks for your feedback. I found [this](https://github.com/mohachouch/forms-wpf-progress/blob/master/Status.md) list of pending features but didn't see anything about GesturePan on it.
